advertising/transaction-message/reference.md

```diff
@@ -147,14 +155,31 @@ Defines a container object that contains details about occupancy.
 |NumAdults|Required.<br />Data type is integer.<br /><br />The maximum number of adults the room is intended to accommodate.|None
 |Children|Optional.<br /><br />A container object the identifies the ages of children the room is intended to accommodate.|[Children Type](#children-type)
 
+## OccupancySettings Type
+
+Defines the settings which can restrict or modify the occupancy requirements of a room.
+
+|Element|Description|Children
+|-|-|-
+|MinOccupancy|Data type is integer.<br /><br />The minimum number of guests that can stay in a room. For example, if set to 2, this room cannot be booked for a single guest. <br /><br />For example, use USD for United States Dollar. |None
```

Comment:
``For example, use USD for United States Dollar.``

This looks like wrong here. Example should be about integer between 2 and 20.

Reply:
fixed copy pasta please verify